frontend/talentsearch/src/js/components/browse/PoolCard.tsx

```diff
@@ -0,0 +1,178 @@
+        locale,
+      );
+    }
+    return undefined;
+  };
+
+  const poolEssentialSkillKeys = (): JSX.Element[] | null => {
+    if (pool.essentialSkills && pool.essentialSkills[0]) {
+      const chipsArray = pool?.essentialSkills?.map((skill) => (
+        <Chip
+          key={`essentialSkill-${skill.id}`}
+          mode="outline"
+          color="neutral"
+          label={getLocalizedName(skill.name, intl)}
+        />
+      ));
+      return chipsArray;
+    }
+    return null;
+  };
+
+  return (
+    <div
+      className="card"
+      data-h2-display="base(block)"
+      data-h2-shadow="base(s)"
+      data-h2-max-width="base(40rem)"
+    >
+      <div
+        className="card__body"
+        data-h2-background-color="base(dt-white)"
+        data-h2-padding="base(x.75)"
+      >
+        <h6
+          data-h2-font-weight="base(700)"
+          data-h2-padding="base(0, 0, x0.75, 0)"
+        >
+          {poolClassificationString()
+            ? poolClassificationString()
+            : intl.formatMessage({
+                defaultMessage: "(No Classification)",
+                description: "No Classification",
+                id: "y+OZjx",
+              })}{" "}
+          {pool.stream ? intl.formatMessage(getPoolStream(pool.stream)) : ""}
+        </h6>
+
+        <p data-h2-padding="base(0, 0, x0.75, 0)">
+          {pool.description ? pool.description[locale] : ""}
+        </p>
+        <p data-h2-padding="base(0, 0, x0.75, 0)">
+          <span
+            data-h2-font-weight="base(700)"
+            data-h2-padding="base(0, x0.25, 0, 0)"
+          >
+            {intl.formatMessage({
+              defaultMessage: "Salary range",
+              id: "MKyZVJ",
+            })}
+            :{" "}
+          </span>{" "}
+          {poolClassificationSalary()
+            ? poolClassificationSalary()
+            : intl.formatMessage({
+                defaultMessage: "(Needs classification)",
+                description: "Needs classification",
+                id: "nTGIfl",
+              })}
+        </p>
+
+        <div
+          data-h2-display="base(flex)"
+          data-h2-flex-direction="base(row)"
+          data-h2-align-items="base(center)"
+          data-h2-padding="base(0, 0, x0.75, 0)"
+        >
+          <span
+            data-h2-font-weight="base(700)"
+            data-h2-padding="base(0, x0.25, 0, 0)"
+          >
+            {intl.formatMessage({
+              defaultMessage: "Key skills",
+              description: "key skills",
+              id: "ppuSbM",
+            })}
+            :{" "}
+          </span>{" "}
+          <span>
+            <Chips>
+              {poolEssentialSkillKeys()
+                ? poolEssentialSkillKeys()
+                : intl.formatMessage({
+                    defaultMessage: "(No skills required)",
+                    description: "No skills required",
+                    id: "WbET08",
+                  })}
+            </Chips>
```

Comment:
`<Chips>` should be moved within the check since it contains `role=list` which should not be empty.

```tsx
{essentialSkills ? <Chips>{essentialSkills}</Chips> : intl.formatMessage(...)}
```